components/chat/chatBar.tsx

```diff
@@ -194,6 +194,7 @@ const ChatBar = ({
                   event.preventDefault();
                   commandsRef.current?.focusCatalog();
                 } else {
+                  event.preventDefault();
```

Comment:
nit: this is now in every branch of this if-statement. Can it be added at the top?